Northstar.Client/mod/scripts/vscripts/spectator_pov_button_fix.nut

```diff
@@ -0,0 +1,12 @@
+global function SpectatorPovButtonFixInit
+
+void function SpectatorPovButtonFixInit(){
+	RegisterButtonPressedCallback(MOUSE_MIDDLE, SwitchPov)
+}
+
+void function SwitchPov(var button){
+	if(IsSpectating() && GetLocalClientPlayer() != null){
+		GetLocalClientPlayer().ClientCommand("spec_mode")
```

Comment:
I need to look into this properly, but isn't spec_mode already bound? If so, why doesn't it already work?

Reply:
Just an update, it's handled in native and is a pain in the ass.
It works changing from first person to third person, but not the other way around?

This conversation could probably be marked as resolved